packages/discord.js/src/structures/InteractionWebhook.js

```diff
@@ -25,32 +27,38 @@ class InteractionWebhook {
   }
 
   // These are here only for documentation purposes - they are implemented by Webhook
-  /* eslint-disable no-empty-function */
+
   /**
    * Sends a message with this webhook.
+   *
    * @param {string|MessagePayload|InteractionReplyOptions} options The content for the reply
    * @returns {Promise<Message>}
    */
 
-  send() {}
+  async send() {}
 
   /**
    * Gets a message that was sent by this webhook.
+   *
    * @param {Snowflake|'@original'} message The id of the message to fetch
    * @returns {Promise<Message>} Returns the message sent by this webhook
    */
 
-  fetchMessage() {}
+  async fetchMessage() {}
 
   /**
    * Edits a message that was sent by this webhook.
+   *
    * @param {MessageResolvable|'@original'} message The message to edit
    * @param {string|MessagePayload|WebhookMessageEditOptions} options The options to provide
    * @returns {Promise<Message>} Returns the message edited by this webhook
    */
 
-  editMessage() {}
+  async editMessage() {}
+
   deleteMessage() {}
```

Comment:
Should this have `async` too?

Reply:
The other methods only need it because their JSDoc states that they return a Promise. For this method (and all the other places you mentioned this) the methods are only there to make jsdoc2md to inherit the JSDoc from parent class.